node-fs-readfile/cat.js

```diff
@@ -0,0 +1,22 @@
+import { readFile } from 'node:fs/promises';
+
+const args = process.argv.slice(2);
+
+const promises = args.map(async (filePath) => {
+  try {
+    const data = await readFile(filePath, {
+      encoding: 'utf8',
+    });
+    console.log(data);
```

Comment:
This approach works, but it logs the files in a random order, depending on when the read finishes. To log them in the order they were specified on the command line, you need to get the values from Promise.all and log those.